### ISSUE TITLE
Add Slack join information

### ISSUE THREAD

**Issue by luxas** [cloud-native-nordics/presentations#4]
Where and how to join the Slack channel

**Comment by Steiniche** [comment]
This should be done after the rename of the channel.
@kaspernissen please give us a heads up or fix this issue when you are doing it!

**Comment by kaspernissen** [comment]
Sure, I just have to make sure that the sign-up process will work after renaming. After that, we should probably be looking into what options is out there for a better Slack sign-up experience.

**Comment by kaspernissen** [comment]
Should we include this information in the README.md for this repository? I added a small note about this on the organization page.

**Comment by luxas** [comment]
Yes, some paragraphs of how to join and why in the README please. Thanks!
